### PR TITLE
Extensions integration

### LINKED ISSUE #5361: Add a basic panel to load/manage remote extensions
### User Story

As a user, I want an easy way to load and manage extensions lifecycle.

### Description

*Type*: Feature

Add a screen in profile/advanced allowing to load extensions. Will be hidden behind a flag and accessible in dev mode only.

An example extension universal link is: https://get.status.im/extension/gist@jeluard/87b2fd97baee988fbf74c442be235fe3

frame

### PR DESCRIPTION
**Pull request by jeluard**
Fixes #5361

### Summary:

Add a hook definition for chat commands and a basic way to load remote extensions.

Hidden behind a flag.

### CODE REVIEW

**Review by janherich — COMMENTED** [review]
I would prefer for my app-state registry changes to land first in pluto, it's just couple of hours of work more and the integration will be much cleaner :)

**Comment by jeluard** [comment]
@janherich Sure let's wait!

**Review by janherich — APPROVED** [review]

**Comment by vitvly** [comment]
@janherich Is there anything left blocking this from merge?

**Comment by jeluard** [comment]
@siphiuel Still WIP AFAIK

File: src/status_im/ui/screens/db.cljs [hunk 1]
```diff
@@ -55,7 +56,8 @@
              :tooltips                           {}
              :desktop/desktop                    {:tab-view-id :home}
              :dimensions/window                  (dimensions/window)
-             :push-notifications/stored          {}})
+             :push-notifications/stored          {}
+             :registry                           {}})
```
Comment (flexsurfer) [inline src/status_im/ui/screens/db.cljs:60]: maybe better to name it something like `:extensions-registry` ?

File: src/status_im/ui/screens/extensions/add/styles.cljs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            [status-im.ui.components.colors :as colors]))
+
+(def wrapper
+  {:flex   1
+   :margin 16})
+
+(def input-container
+  {:flex-direction    :row
+   :align-items       :center
+   :justify-content   :space-between
```
Comment (flexsurfer) [inline src/status_im/ui/screens/extensions/add/styles.cljs]: extra spaces

File: src/status_im/ui/screens/extensions/add/styles.cljs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+   :border-radius     styles/border-radius
+   :height            52
+   :margin-top        15})
+
+(defstyle input
+  {:flex               1
+   :font-size          15
+   :letter-spacing     -0.2
```
Comment (flexsurfer) [inline src/status_im/ui/screens/extensions/add/styles.cljs]: extra spaces

**Review by flexsurfer — APPROVED** [review]
Awesome work guys!

**Pull request merged**